cmd/svcat/binding/unbind_cmd.go

```diff
@@ -134,6 +136,10 @@ func (c *unbindCmd) unbindInstance() error {
 			}(binding.Namespace, binding.Name)
 		}
 		g.Wait()
+	} else {
+		for _, binding := range bindings {
+			output.WriteDeletedResourceName(c.Output, binding.Name)
+		}
```

Comment:
What if some bindings can't be deleted and some can? In that case, the method never gets to here because it immediately returns the error. The user sees only the error and there's no indication that some bindings were in fact deleted.

That was the reason I added this weird looking if statement: 
https://github.com/luksa/service-catalog/blob/97296b455101551cf52c52704439a1fcf2e3d525/cmd/svcat/binding/unbind_cmd.go#L107-L109

Reply:
Good catch, thank you! I'll get that updated.